assets/label_tools/edit_labels.py

```diff
@@ -0,0 +1,45 @@
+import cv2, os, json
+
+
+DIR = os.path.dirname(os.path.abspath(__file__))
+LABELS_IN = '/input/labels.json'
+LABELS_OUT = '/data/image_labels.json'
+IMAGE_IN = '/input/image-%d.jpg'
+IMAGE_OUT = '/data/images/%d.jpg'
+
```

Comment:
Might be worth making these into run-time switch variables so that  people can organize their datasets and output directories as they please

Reply:
I usually run this from the IDE. Also, this snippet is not part of the pipeline.
With that said, I can add the command line functionality if there is an option to still run from IDE with default values. Do you have suggestions?

Reply:
Could do something like: 

ap = argparse.ArgumentParser()
ap.add_argument("-i", "--IMAGE_IN", required=False,
	help="path to the input image")
args = vars(ap.parse_args())
if !(args[IMAGE_IN]):
  IMAGE_IN = '/input/image-%d.jpg'

Reply:
But this might be overkill, user just has to be aware that they might need to change the location of the variables before executing. I'm ok with how it is

Reply:
Actually nvm, a bit too overkill